Mudlet/Mudlet
# Fix: Show Client.GUI GMCP in Lua interpreter

#### Brief overview of PR changes/additions
The outcome desired is 1) for Client.GUI GMCP messages to process that loads a GUI provided by the game, and 2) show the GMCP output rendered in the Lua interpreter.  Currently, there are `return;` statements blocking the GMCP messages from being passed to the Lua interpreter and showing up in the client after the processing.

#### Motivation for adding to Mudlet
A discussion on the [Mudlet Discord](https://discord.com/channels/283581582550237184/283582068334526464/1073397883367342121).

#### Other info (issues closed, discussion etc)
Does not close #6540 however it should be merged before that PR is done.

How to test:

1. If necessary, remove any StickMUD profile you have right now and start with a fresh profile (stickmud.com 7670 check secure)
2. Login to StickMUD with a new character, choosing to use the provided GUI during character generation.  After the UI loads, verify Client.GUI got to the Lua interpreter by choosing `lua gmcp.Client`.
3. Close and restart Mudlet, login with the same character (the UI should be showing).
4. Verify, again, that the Client.GUI got to the Lua interpreter by choosing `lua gmcp.Client`.

If this above was run in 4.16 the GMCP does not show in the Lua interpreter when sent as its own GMCP message.
